content/c-sharp/concepts/strings/strings.md

```diff
@@ -0,0 +1,190 @@
+---
+Title: 'Strings'
+Description: 'A string is an instance of the System.String object in C# and represents an immutable sequence of Char objects.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Data Types'
+  - 'Strings'
+  - 'Objects'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+A **`string`** is an instance of the `System.String` object in C# and represents an immutable sequence of `Char` objects. Unlike other languages, there's no null-terminating character at the end of a C# string, so a C# string can contain embedded null characters. The `string` keyword in C# is an alias for `System.String` and can be used even without `using System;`.
+
+> **Note:** Strings are immutable, which means they do not change once created. Methods and operators that appear to change a string in a variable really assign a new string to the variable and release the old string for garbage collection. This is important to realize when using references, since a reference pointing to a string that's subsequently "modified" will continue to point to the original unmodified string.
+
+## Declaring and Initializing
+
+There are many different ways to declare and initialize a string in C#. The following example lists a number of them:
+
+```cs
+// Declare without initializing.
+string myString1;
+
+// Declare using System.String.
+System.String myString2;
+
+// Initialize with a string literal.
+string myString3 = "Hello World!";
+
+// Local variables can use implicit typing.
+var localString = "Still a strongly-typed string.";
+
+// The string constructor can create a string from a char*, char[] or sbyte*.
+char source = {'1','2','3','A','B','C'};
+string myString4 = new string(source);
+```
+
+The `new` operator is only used to create a string when initializing with an array of `char`.
+
+> **Note:** It is best practice to initialize an empty string with the `System.String.Empty` constant value or the `""` string literal rather than with `null` to avoid an unwanted `NullRefrenceException`.
```

Comment:
```suggestion
> **Note:** It is best practice to initialize an empty string with the `System.String.Empty` constant value or the `""` string literal rather than with `null` to avoid an unwanted `NullReferenceException`.
```